src/olympia/amo/tests/test_utils_.py

```diff
@@ -172,3 +175,19 @@ def bar(self):
     del foo.bar
     assert foo.bar == 'original value'
     assert callme.call_count == 1
+
+
+@pytest.mark.parametrize('lang', settings.AMO_LANGUAGES)
+def test_get_locale_from_lang(lang):
+    """Make sure all languages in settings.AMO_LANGUAGES can be resolved."""
+    locale = get_locale_from_lang(lang)
+
+    assert isinstance(locale, Locale)
+    assert locale.language == lang[:2]
```

Comment:
`get_locale_from_lang` is used exactly two times, in our `amo.helpers` and here it's passed directly to babel. And the other one is in `stats.models` and passes it also directly to babel.

I think we're safe here. We only pass the locale forward to babel and don't use it ourselfs. It could fix some things, e.g formattings or make them more correct maybe - which I doubt - but that's the only thing I can imagine.